src/chi/components/alert/_bubble.scss

```diff
@@ -3,6 +3,7 @@
 .chi {
   .m-alert {
     &.-bubble {
+      align-items: center;
 
       & > .-close {
         position: absolute;
```

Comment:
@davidojedacl try to avoid the position absolute here and wrap the icon and the text in a div. So you can use flex properties properly